### DISCUSSION TITLE
aws subdirectory

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#11222] [Archive]
how to run directus in subdirectory on aws like www.example.com/directus/ by using port 443

**Comment by azrikahar** [comment]
It's best not to open similar/duplicate discussions, but [the reply to your previous question](https://github.com/directus/directus/discussions/11223#discussioncomment-2032456) is still applicable here 👍 You can also check out other similar discussion with similar answers such as:

- https://github.com/directus/directus/discussions/3307
- https://github.com/directus/directus/discussions/4480
- https://github.com/directus/directus/discussions/9152
